images/php/php-nginx/Debian/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
+    echo "Hello, World!";
```

Comment:
Could add an error_log message to test error logs go to right place

`error_log("TEST")`